.changeset/mighty-tips-attend.md

```diff
@@ -0,0 +1,55 @@
+---
+'@jpmorganchase/mosaic-theme': patch
```

Comment:
lgtm, but I think this may constitute more than a patch?

Reply:
we are on beta numbering at the mo, so we have just been using patch for everything

Reply:
Agreed...updated the changset